File: backend/internal/models/telegram_send_log.go
package models

import (
	"time"

	"github.com/uptrace/bun"
)

type TelegramSendLogStatus string

const (
	StatusSent   TelegramSendLogStatus = "SENT"
	StatusFailed TelegramSendLogStatus = "FAILED"
)

type TelegramSendLog struct {
	bun.BaseModel `bun:"table:telegram_send_log"`

	ID      int64                 `bun:"id,pk,autoincrement" json:"id"`
	ShopID  int64                 `bun:"shop_id,notnull" json:"shop_id"`
	OrderID int64                 `bun:"order_id,notnull" json:"order_id"`
	Message string                `bun:"message,notnull" json:"message"`
	Status  TelegramSendLogStatus `bun:"status,notnull" json:"status"`
	Error   *string               `bun:"error" json:"error,omitempty"`
	SentAt  time.Time             `bun:"sent_at,notnull,default:current_timestamp" json:"sent_at"`
}

File: backend2/src/Entity/TelegramSendLog.php
<?php

namespace App\Entity;

use App\Repository\TelegramSendLogRepository;
use Doctrine\ORM\Mapping as ORM;
use Doctrine\DBAL\Types\Types;

#[ORM\Entity(repositoryClass: TelegramSendLogRepository::class)]
#[ORM\Table(name: 'telegram_send_log')]
class TelegramSendLog
{
    public const STATUS_SENT = 'SENT';
    public const STATUS_FAILED = 'FAILED';

    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column]
    private ?int $shopId = null;

    #[ORM\Column]
    private ?int $orderId = null;

    #[ORM\Column(type: Types::TEXT)]
    private ?string $message = null;

    #[ORM\Column(length: 10)]
    private ?string $status = null;

    #[ORM\Column(type: Types::TEXT, nullable: true)]
    private ?string $error = null;

    #[ORM\Column(type: Types::DATETIME_MUTABLE)]
    private ?\DateTimeInterface $sentAt = null;

    public function __construct()
    {
        $this->sentAt = new \DateTime();
    }

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getShopId(): ?int
    {
        return $this->shopId;
    }

    public function setShopId(int $shopId): static
    {
        $this->shopId = $shopId;

        return $this;
    }

    public function getOrderId(): ?int
    {
        return $this->orderId;
    }

    public function setOrderId(?int $orderId): static
    {
        $this->orderId = $orderId;

        return $this;
    }

    public function getMessage(): ?string
    {
        return $this->message;
    }

    public function setMessage(string $message): static
    {
        $this->message = $message;

        return $this;
    }

    public function getStatus(): ?string
    {
        return $this->status;
    }

    public function setStatus(string $status): static
    {
        $this->status = $status;

        return $this;
    }

    public function getError(): ?string
    {
        return $this->error;
    }

    public function setError(?string $error): static
    {
        $this->error = $error;

        return $this;
    }

    public function getSentAt(): ?\DateTimeInterface
    {
        return $this->sentAt;
    }

    public function setSentAt(\DateTimeInterface $sentAt): static
    {
        $this->sentAt = $sentAt;

        return $this;
    }
}
